<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mirror</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-image:url(&quot;./ressources/background_1280_720.png&quot;)
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="scanNetwork">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>186</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Scan Network</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Target IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
     font: bold 14px;
	 color: white;
 }</string>
    </property>
    <property name="text">
     <string>Console output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="launchArpAttack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Launch ARP attack</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetMAC">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Target MAC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="routerIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>610</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Router IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLabel {
     font: bold 14px;
	 color: white;
 }</string>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputZone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>301</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopArpAttack">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>640</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Stop ARP attack</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputSniffing">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>891</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
